example/lib/shared/npc/critter/critter.dart

```diff
@@ -15,30 +15,22 @@ class Critter extends SimpleNpc
           position: position,
           size: Vector2.all(DungeonMap.tileSize * 0.8),
           speed: DungeonMap.tileSize * 2.75,
```

Comment:
I think it's a bit too fast, it gets glued to the player and the animation doesn't play properly because of it.